transport-native-io_uring/src/main/java/io/netty/channel/uring/Native.java

```diff
@@ -32,7 +32,7 @@
 final class Native {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(Native.class);
     static final int DEFAULT_RING_SIZE = Math.max(64, SystemPropertyUtil.getInt("io.netty.uring.ringSize", 4096));
-    static final boolean DEFAULT_USE_IOSEQ_ASYNC = true;
+    static final int DEFAULT_IOSEQ_ASYNC_THRESHOLD = 25;
```

Comment:
this was the number after which using the flag showed good results here...

Reply:
let's use a sys prop here? That would be nice to help with tests

Reply:
sure why not